cmake/common/targets/clang-tidy.cmake

```diff
@@ -35,13 +35,11 @@ function(sourcemeta_target_clang_tidy)
   file(GLOB_RECURSE SOURCEMETA_TARGET_CLANG_TIDY_FILES
     ${SOURCEMETA_TARGET_CLANG_TIDY_SOURCES})
 
-  set(CLANG_TIDY_CONFIG "${CMAKE_CURRENT_FUNCTION_LIST_DIR}/clang-tidy.config")
```

Comment:
Instead of removing this, can you tweak the variable to point at `${PROJECT_DIR}/.clang-tidy` (or whatever the right variable name is?)